kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/test/resources/various-fields-taskform.json

```diff
@@ -69,6 +69,22 @@
 						}
 					]
 				}
+			},
+			{
+				"name": "startDate_",
+				"typeInfo": {
+					"type": "BASE",
+					"className": "java.time.LocalDateTime",
```

Comment:
Trying to understand here - so LocalDate (as reported in the issue) is internally mapped to LocalDateTime and processes as such, right?

Reply:
Again, the issue is about a LocalDateTime